core/server/common/src/main/java/alluxio/web/entity/PaginationOptionsEntity.java

```diff
@@ -0,0 +1,95 @@
+/*
+ * The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+ * (the "License"). You may not use this work except in compliance with the License, which is
+ * available at www.apache.org/licenses/LICENSE-2.0
+ *
+ * This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied, as more fully set forth in the License.
+ *
+ * See the NOTICE file distributed with this work for information regarding copyright ownership.
+ */
+
+package alluxio.web.entity;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * The web page grid pagination options entity.
+ */
+public class PaginationOptionsEntity {
```

Comment:
You mean I should mark the class `final` because of none subclass will extends it? I think so, I will do it.